src/main/java/de/unistuttgart/iste/gits/content_service/controller/ContentController.java

```diff
@@ -70,7 +70,7 @@ public MediaContent updateMediaContent(@Argument UpdateMediaContentInput input,
     }
 
     @MutationMapping
-    public Assessment createAssessment(@Argument CreateAssessmentInput input, @Argument UUID courseId) {
+    public Assessment _internal_createAssessment(@Argument CreateAssessmentInput input, @Argument UUID courseId) {
```

Comment:
This is not the Java naming convention. Please keep the old name and specify the correct name in the MutationMapping annotation

Reply:
@Paul I did it